forge-gui/res/cardsfolder/upcoming/altair_ibn_laahad.txt

```diff
@@ -0,0 +1,15 @@
+Name:Altaïr Ibn-La'Ahad
+ManaCost:R W B
+Types:Legendary Creature Human Assassin
+PT:3/3
+K:First Strike
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ Whenever CARDNAME attacks, exile up to one target Assassin creature card from your graveyard with a memory counter on it. Then for each creature card you own in exile with a memory counter on it, create a tapped and attacking token that’s a copy of it. Exile those tokens at end of combat.
+SVar:TrigExile:DB$ ChangeZone | Imprint$ True | ValidTgts$ Creature.Assassin+YouCtrl | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target Assassin creature card from your graveyard | Origin$ Graveyard | Destination$ Exile | SubAbility$ DBPutCounter
+SVar:DBPutCounter:DB$ PutCounter | CounterType$ MEMORY | CounterNum$ 1 | Defined$ Imprinted | SubAbility$ DBCopyPermanent
```

Comment:
use WithCountersType instead

Reply:
Done. More items to note down for cleanup I take it.